src/index.js

```diff
@@ -251,7 +293,10 @@ function getItemValues(item, key) {
     // handle nested keys
     value = key
       .split('.')
-      .reduce((itemObj, nestedKey) => itemObj ? itemObj[nestedKey] : null, item)
+      .reduce(
+        (itemObj, nestedKey) => (itemObj ? itemObj[nestedKey] : null),
+        item,
+      )
```

Comment:
No changes here, the linter wanted it like this :)

Reply:
I think the last individual to contribute to this project didn't install the dependencies so this wasn't automatically formatted 😅